src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -250,6 +252,23 @@ public Architecture TargetPlatform
         }
     }
 
+    /// <summary>
+    /// Gets or sets the default platform that will be used for AnyCPU sources, or non-dll sources. Possible values are <see cref="Architecture"/> except for AnyCPU or Default.
+    /// </summary>
+    public Architecture? DefaultPlatform
```

Comment:
Should we name it better like AnyCPU[Concrete/...]Platform?

Reply:
It is not just for AnyCPU, but for any source that does not determine a platform. best names are imho FallbackPlatform or DefaultPlatform, which is what I used.